include/cutlass/epilogue/collective/sm90_epilogue_tma_warpspecialized.hpp

```diff
@@ -600,7 +612,14 @@ class CollectiveEpilogue<
         }
 
         // Copy tile from register to smem
-        copy(tiled_r2s, tRS_rD, tRS_sD(_,_,_,store_pipe_producer_state.index()));
+        [&]() -> void {
+          if constexpr (DispatchPolicy::EnableNullptrD) {
+            if (params.is_nullptr_d) {
```

Comment:
is a dynamic branch needed here? looks like you always check against the template parameter only. So we can totally avoid any dynamic checks within the kernel which will ensure no perf regressions.

Reply:
I followed the code pattern in Sm90AuxStore. I think there are two possible implementations here. 1. use EnableNullptrD = true as default and dynamic check; 2. use DisableWriteD = false as default and constant check; I think It makes sense to prefer the second one for D.

Reply:
I have updated to use static branches. Could you please take a look?